src/gui/MItem_lan.hpp

```diff
@@ -37,9 +37,9 @@ class MI_WIFI_CREDENTIALS_t : public WI_LABEL_t {
 };
 
 class MI_NET_INTERFACE_t : public WI_SWITCH_t<3> {
-    constexpr static const char *const label = "Network Interface"; //do not translate
+    constexpr static const char *const label = "Interface"; //do not translate
 
-    constexpr static const char *str_off = N_("None");
+    constexpr static const char *str_off = "Off";    //do not translate
```

Comment:
Just a small question: why should this not be translated?

Reply:
Off is multinational

Reply:
Or at least every body knows what it means